sdk/search/search-documents/src/searchServiceClient.ts

```diff
@@ -163,6 +170,14 @@ export class SearchServiceClient {
     this.client = new GeneratedClient(dummyCredential, this.apiVersion, this.endpoint, pipeline);
   }
 
+  /**
+   * Retrieves the search index client corresponding to this search service client
```

Comment:
do we English-ify the type names normally in comments? Or should this just reference `SearchIndexClient` and `SearchServiceClient` directly? /cc @bterlson

Reply:
Changed it